openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/StreamProcessor.java

```diff
@@ -188,6 +188,10 @@ private void handleBroadcastStarted(Client c, final String uid, JSONObject msg)
 			log.error("Failed to start broadcast", e);
 		}
 	}
+	
+	public KStream startBroadcast(KStream stream, StreamDesc sd, String sdpOffer) {
```

Comment:
Unfortunate I had to modify the main code to write a unit test. But that seemed the only way how I can prevent calling the webRTC methods in KStream. Cause otherwise mocking those is almost impossible.

Reply:
please add method comments so this method will not be refactored